frontend/src/features/account/IENotSupportedPage.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Col, Container, Row } from 'react-bootstrap';
+
+export const IENotSupportedPage: React.FC = () => {
```

Comment:
We're trying to have top level component jsdoc for all new components, so please add some here.